ee/clickhouse/queries/funnels/test/test_funnel.py

```diff
@@ -43,7 +43,27 @@ def _create_event(**kwargs):
 
 class TestFunnelBreakdown(ClickhouseTestMixin, funnel_breakdown_test_factory(ClickhouseFunnel, ClickhouseFunnelPersons, _create_event, _create_person)):  # type: ignore
     maxDiff = None
-    pass
+
+    def test_funnel_query_with_denormalized_breakdown(self):
```

Comment:
I think it's worth adding this to `funnel_breakdown_test_factory` instead? It deals with all Clickhouse orderings.

Reply:
Done!